lona/default_settings.py

```diff
@@ -57,6 +57,7 @@
 # views
 CORE_FRONTEND_VIEW = 'lona.default_views.FrontendView'
 FRONTEND_VIEW = ''
+INITIAL_STATE: Dict[Any, Any] = {}
```

Comment:
Just use ``dict`` here. Can you rename ``INITIAL_STATE`` to ``INITIAL_SERVER_STATE``?

Reply:
Updated. Please take another look at the type annotations :-)